Lesson1/Task4/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+
+namespace Task4
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Enter first row: ");
+            string row1 = Console.ReadLine();
+
+            Console.Write("Enter second row: ");
+            string row2 = Console.ReadLine();
+
+            Console.Write("Enter character: ");
+            string ch = Console.ReadLine();
+
+            string conc = row1 + row2;
+            int index = conc.LastIndexOf(ch);
+
+            Console.WriteLine();
+            Console.WriteLine("Index of " + ch + " in " + conc + ": " + index);

```

Comment:
@okuzmina Better changed "Index" to "Last index" because, there could be multiple times ch found in string, also string format, as above